### DISCUSSION TITLE
Server maintenance mode

### DISCUSSION THREAD

**Discussion by neyunse** [coollabsio/coolify#2949] [Feature Requests]
I don't know if it would be interesting but it might be useful. For example in the server tab add an option to activate a maintenance mode where all web projects that are linked show a warning in case the server is under maintenance or only the sites that we want to show this warning. Deployments will still be able to run in the background to keep the projects up to date, but will only be visible after the maintenance is deactivated.

**Comment by ScarVite** [comment]
it'd definitely be something that would be to use for us.

Some of our internal users like still working on our system while they know we are performing an update and then complain why its slow/unresponsive.

so having a "maintenance" page shown would be useful

**Comment by 5ynq** [comment]
IMO, it's really worth it!

**Comment by KillerCodeMonkey** [comment]
push :)

Especially when there is postgres 18 released. And migration could be take some time.
It would be nice - instead of adding some custom maintenance pages or introduce similar approaches - to have a maintenance mode for a server, so it can be toggled on and off real fast.

**Comment by AMichaelSuess** [comment]
Would be nice to have indeed, also while cloning resources and moving them between destinations.
